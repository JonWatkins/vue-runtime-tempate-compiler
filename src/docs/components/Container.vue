<template>
  <div class="container-fluid">
    <div class="row flex-xl-nowrap">
      <slot />
    </div>
  </div>
</template>

<script>
export default {
  name: 'Container'
}
</script>
