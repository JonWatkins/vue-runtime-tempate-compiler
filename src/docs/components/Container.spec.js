import { shallowMount } from '@vue/test-utils'
import Container from './Container.vue'
const { localVueInst } = global

let wrapper

describe('Container', () => {
  beforeEach(() => {
    wrapper = shallowMount(Container, {
      localVue: localVueInst
    })
  })

  it('is called', () => {
    expect(wrapper.name()).toBe('Container')
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
