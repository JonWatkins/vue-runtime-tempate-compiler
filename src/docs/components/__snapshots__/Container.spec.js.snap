// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container render correctly 1`] = `
<div class="container-fluid">
  <div class="row flex-xl-nowrap"></div>
</div>
`;
